chart/values.yaml

```diff
@@ -47,6 +47,8 @@ gateway:
     pullPolicy: IfNotPresent
   errorPageURL: https://www.integration.signin.service.gov.uk/proxy-node-error
   metadataPath: /ServiceMetadata
+  qwac:
+    enabled: false
```

Comment:
could we move this up to `global`, as this switch may be used for connector node metadata hosting, which may not be in gateway.

Can we change the key to `qwacCertificate` to make it a bit clearer? 🤷‍♂️

Reply:
I don't suppose it will take up much more of the RAM-Memory so it should be fine.  (Sorry, I'm a pedant when it comes to that sort of thing, I can change it to qwacCertificate)